packages/components/src/combobox-control/index.js

```diff
@@ -68,7 +68,9 @@ function ComboboxControl( {
 		const containsMatch = [];
 		const match = inputValue.toLocaleLowerCase();
```

Comment:
Should we deburr this too?

Reply:
Yes, check the new commit 7e41bcb746878bc727897ac1661e0fc7ee824d87